src/screens/FeedScreen/RenderList.js

```diff
@@ -296,7 +312,9 @@ RenderListFeed.propTypes = {
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
   onHeaderOptionClicked: PropTypes.func,
-  onPreviewCommentPress: PropTypes.func
+  onPreviewCommentPress: PropTypes.func,
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New PropTypes have been added for the new props. This is good for type checking, but make sure to also add these new props to the defaultProps if they are not required.

```diff
+  onPreviewCommentPress: PropTypes.func,
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->